crates/asm/txs/bridge-v1/src/deposit_request/aux.rs

```diff
@@ -53,29 +81,91 @@ impl DrtHeaderAux {
     }
 }
 
+impl Codec for DrtHeaderAux {
```

Comment:
Why do we need the manual impl?
Can we just keep using the `#[derive(Codec)]`?
Is this because one of the new fields does not impl `Codec`?
If so, should we add the derive to this new field, and then keep the derive here?